.circleci/config.yml

```
@@ -0,0 +1,159 @@
+# .circleci/config.yml
+version: 2.1
+
+executors:
+  # osx-arm64:
+  #   macos:
+  #     xcode: 14.2.0 # indicate your selected version of Xcode
+  #   resource_class: macos.m1.large.gen1
+  linux-aarch64:
+    docker:
+      - image: cimg/base:2023.06
+    resource_class: arm.medium
+
+jobs: # a basic unit of work in a run
+  build_and_test:
+    parameters:
+      os:
+        type: executor
+    executor: << parameters.os >>
+    steps:
+      - checkout
+
+      - run:
+          name: Fetch bioconda install script
+          command: wget https://raw.githubusercontent.com/bioconda/bioconda-common/master/{common,install-and-set-up-conda,configure-conda}.sh
+
+      - restore_cache:
+          keys:
+            - bioconda-utils-{{ arch }}-{{ checksum "install-and-set-up-conda.sh" }}-{{ checksum "configure-conda.sh" }}-{{ checksum "common.sh" }}
+
+      - run:
+          name: Install bioconda-utils
+          command: |
+            sudo mkdir -p /opt/
+            sudo chmod o+rwx /opt
+            bash install-and-set-up-conda.sh
+
+      - save_cache:
+          key: bioconda-utils-{{ arch }}-{{ checksum "install-and-set-up-conda.sh" }}-{{ checksum "configure-conda.sh" }}-{{ checksum "common.sh" }}
+          paths:
+            - /opt/mambaforge/
+    
+      - run:
+          name: Setup PATH
+          command:
+            echo 'export PATH=/opt/mambaforge/bin:"$PATH"' >> "$BASH_ENV"
+
+      # We reconfigure conda to use the right channel setup.
+      # This has to be done after the cache is restored, because
+      # the channel setup is not cached as it resides in the home directory.
+      # We could use a system-wide (and therefore cached) channel setup,
+      # but mamba does not support that at the time of implementation
+      # (it ignores settings made by --system).
+      - run:
+          name: Configure conda
+          command: bash configure-conda.sh
+
+      - run:
+          name: Build and test
+          command: |
+            source /opt/mambaforge/etc/profile.d/conda.sh
+            source /opt/mambaforge/etc/profile.d/mamba.sh
+            mamba activate bioconda
+            bioconda-utils build recipes config.yml --git-range origin/master HEAD
+
+      - run:
+          name: Prepare artifacts
+          command: |
+            (
+            mkdir -p /tmp/artifacts/packages
+            cd /opt/mambaforge/envs/bioconda/conda-bld || exit 0
+            find -name .cache | xargs rm -rf || true
+            for n in index.html channeldata.json linux-64 osx-64 noarch; do
```

@johanneskoester I'm not seeing the `linux-aarch64` artifacts in CircleCI, only json and html files for `noarch`. I am think this is probably where it is missing.

```suggestion
            for n in index.html channeldata.json linux-aarch64 linux-64 osx-64 noarch; do
```

---

Since it's minor, I'm going to got ahead and commit this change to see if it produces the artifacts so I can test the artifact download.